### DISCUSSION TITLE
Question: I'd like to create a mobile client for this project.

### DISCUSSION THREAD

**Discussion by gsmlg** [semaphoreui/semaphore#1746] [Show and tell]
I'd like to create a mobile client for this project.
How should I contribute?

**Comment by ansibleguy** [comment]
As you are providing no information - please at least add some context:
* Would Web-Frontend support for mobile not be enough? (_single codebase is easier to maintain_)
  * What would be the benefits of a dedicated mobile app?
* What functionality would you want to cover with such an app?

**Comment by gsmlg** [comment]
![WeChatWorkScreenshot_a98d67f8-4180-46c5-87b0-8331532fcd8b](https://github.com/ansible-semaphore/semaphore/assets/3713684/c3854151-7a09-4218-8d75-8479257a0b43)

A mobile app create in flutter.

**Comment by ansibleguy** [comment]
I personally would create a repository under your github account and when the App has 'matured' the maintainer might fork it.
This seems to be the way he has done it with some other sub-projects.
See: https://github.com/semaphoreui

**Comment by gsmlg** [comment]
This App will publish in a few weeks.

**Comment by gsmlg** [comment]
App has been published to Google PlayStore

**Comment by gsmlg** [comment]
> 

@ansibleguy 

https://github.com/gsmlg-dev/ansible-semaphore-client
This is it.

**Comment by ansibleguy** [comment]
FYI @fiftin
